unimath-symbols.ltx

```diff
@@ -4,10 +4,10 @@
 % SYMBOLS DEFINED BY UNICODE-MATH
 %%%%%%%%%%%%%%%%%%%%%%%
 
-\documentclass{article}
+\documentclass[final]{article}
 \makeatletter
 
-\usepackage{booktabs,catchfile,shortvrb,geometry,metalogo,varwidth,textcomp,longtable,hyperref}
+\usepackage{ragged2e,setspace,booktabs,catchfile,shortvrb,geometry,metalogo,textcomp,longtable,tabu,hyperref}
```

Comment:
How necessary is tabu here? I'm happy to use it for this document but I'm wary of using it for future work given it hasn't been updated for some time and it might have some underlying issues: https://tex.stackexchange.com/questions/121841/is-the-tabu-package-obsolete

Having said that, I'm aware my own packages have underlying issues and haven't been updated for a long time, so I'm not one to talk!